Gruntfile.js

```diff
@@ -236,7 +236,7 @@ module.exports = function (grunt) {
                 cmd: '"build/cache/linux32/<%= nwjs.options.version %>/nw" .'
             },
             linux64: {
-                cmd: '"build/cache/linux64/<%= nwjs.options.version %>/nw" .'
+                cmd: '"build/' + projectName + '/linux64/' + projectName + '" .'
             },
```

Comment:
i don't why its the path generated on my system. can you check with a fresh npm install ??